content/millennium/r4/clinical/diagnostics/diagnostic-report.md

```diff
@@ -76,32 +81,25 @@ _Implementation Notes_
 
```

Comment:
since it wasn't changed, it won't let me comment on the line, but if you supply `date` twice, you MUST have prefixes on both values

valid:
date=2023-10-01
date=ge2023-10-01
date=ge2023-10-01&date=lt2023-10-10

not valid:
date=2023-10-01&date=lt2023-10-10
date=2023-10-01&date=2023-10-10

Reply:
Fixing this nowish.  There'll be a new commit shortly